### PR TITLE
DBZ-4011 Surge preview teardown

### PR DESCRIPTION
**Pull request by uidoyen**
Pending surge.sh preview teardown action https://issues.redhat.com/browse/DBZ-4011

### CODE REVIEW

File: .github/workflows/surge_preview_teardown.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Surge.sh Preview Teardown
+
+on:
+  pull_request_target:
```
Comment (gunnarmorling) [inline .github/workflows/surge_preview_teardown.yml:4]: Wouldn't `pull_request` work as well?

File: .github/workflows/surge-preview.yml [hunk 2]
```diff
@@ -31,14 +31,14 @@ jobs:
       - name: upload surge service
         id: deploy
         run: |
-          export DEPLOY_DOMAIN=https://debezium-github-io-preview-pr-${{ steps.pr.outputs.id }}.surge.sh
+          export DEPLOY_DOMAIN=https://debezium-debezium-github-io-preview-pr-${{ steps.pr.outputs.id }}.surge.sh
```
Comment (gunnarmorling) [inline .github/workflows/surge-preview.yml:34]: Are you sure about this? Because the links seem to work in the current form?

**Comment by gunnarmorling** [comment]
@Naros, I believe you set up the Surge account originally, right? Once this PR has been merged, can you drop all the existing preview environments for any non-pending PRs (i.e. any PRs who got closed before this one, causing the preview env to be left over). Thx!

File: .github/workflows/surge-preview.yml (hunk 2 above)
Comment (uidoyen) [reply .github/workflows/surge-preview.yml:34]: Deploying it manually as it is not using `afc163/surge-preview@v1` action here.

**Review by gunnarmorling — APPROVED** [review]

**Comment by uidoyen** [comment]
> @Naros, I believe you set up the Surge account originally, right? Once this PR has been merged, can you drop all the existing preview environments for any non-pending PRs (i.e. any PRs who got closed before this one, causing the preview env to be left over). Thx!

@gunnarmorling I can take care of this once the PR is merged.

**Comment by gunnarmorling** [comment]
Excellent, thanks!

**Pull request merged**